typings/vscode-mssql.d.ts

```diff
@@ -772,4 +769,466 @@ declare module 'vscode-mssql' {
 
 		options: { [name: string]: any };
 	}
+
+	/**
+	 * Namespace for Azure APIs
+	 */
+	export namespace azure {
+
+		/**
+		 * Namespace for Azure Subscriptions
```

Comment:
Could you also link to the source typings file you got these from so we can update them more easily later on?